Example/IntegrationTests/Chat/ChatTests.swift

```diff
@@ -18,23 +18,23 @@ final class ChatTests: XCTestCase {
         inviter = makeClient(prefix: "🍄 Inviter")
     }
 
-    private func waitClientsConnected() async {
```

Comment:
maybe instead of this method `waitClientsConnected` we could wait for clients connected in `setUp` function
look for sign integration tests for reference
but not super important